src/SFA.DAS.EmployerDemand.Data/Configuration/ProviderInterest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace SFA.DAS.EmployerDemand.Data.Configuration
+{
+    public class ProviderInterest: IEntityTypeConfiguration<Domain.Entities.ProviderInterest>
+    {
+        public void Configure(EntityTypeBuilder<Domain.Entities.ProviderInterest> builder)
+        {
+            builder.ToTable("ProviderInterest");
+            builder.HasKey(x => new { x.EmployerDemandId , x.Ukprn }).HasName("PK_ProviderStandard");
```

Comment:
we need a key on this as well to group them together. The Id from the cache on the web should hold the collection of the demands that have been requested, along with all the other data - otherwise when it comes to looking at what the provider did on a dashboard it will be tricky to tie these back to a single action